wp-content/themes/core/assets/css/src/theme/color/_variables.pcss

```diff
@@ -6,48 +6,75 @@
 
 :root {
 	/* -----------------------------------------------------------------------------
-	 * Primary Palette
+	 * Primary: Blue
 	 * ----------------------------------------------------------------------------- */
 
-	--color-primary: #21a6cb;
-	--color-secondary: darken(var(--color-primary), 0.08);
+	--color-primary-10: rgba(217, 244, 255, 1);
+	--color-primary-30: rgba(67, 176, 255, 1);
+	--color-primary-50: rgba(3, 134, 255, 1);
+	--color-primary-70: rgba(11, 95, 222, 1);
+
+	/* -----------------------------------------------------------------------------
+	 * Secondary: Green
+	 * ----------------------------------------------------------------------------- */
+
+	--color-secondary-50: rgba(36, 194, 118, 1);
+	--color-secondary-70: rgba(30, 161, 98, 1);
+
+	/* -----------------------------------------------------------------------------
+	 * Neutral: Grey
+	 * ----------------------------------------------------------------------------- */
+
+	--color-neutral-0: rgba(255, 255, 255, 1);
+	--color-neutral-20: rgba(200, 200, 200, 1);
+	--color-neutral-30: rgba(169, 170, 170, 1);
+	--color-neutral-50: rgba(105, 105, 105, 1);
+	--color-neutral-70: rgba(64, 64, 64, 1);
+	--color-neutral-90: rgba(21, 21, 21, 1);
+
+	/* -----------------------------------------------------------------------------
+	 * Primary
+	 * ----------------------------------------------------------------------------- */
+
+	--color-primary: var(--color-primary-50);
+	--color-secondary: var(--color-secondary-50);
 
 	/* -----------------------------------------------------------------------------
 	 * Whites
 	 * ----------------------------------------------------------------------------- */
 
-	--color-white: #fff;
+	--color-white: var(--color-neutral-0);
 
 	/* -----------------------------------------------------------------------------
 	 * Greys
 	 * ----------------------------------------------------------------------------- */
 
-	--color-grey-light: lighten(var(--color-grey), 0.15);
-	--color-grey: grey;
-	--color-grey-dark: darken(var(--color-grey), 0.15);
+	--color-grey-light: var(--color-neutral-20);
+	--color-grey: var(--color-neutral-50);
+	--color-grey-dark: var(--color-neutral-70);
 
 	/* -----------------------------------------------------------------------------
 	 * Blacks
 	 * ----------------------------------------------------------------------------- */
 
-	--color-black: #000;
-	--color-black-a-40: rgba(0, 0, 0, 0.4);
+	--color-black: var(--color-neutral-90);
 
 	/* -----------------------------------------------------------------------------
 	 * Backgrounds
 	 * ----------------------------------------------------------------------------- */
 
 	--color-bgd: var(--color-white);
 	--color-bgd-content: var(--color-white);
-	--color-bgd-login: #f1f1f1;
+	--color-bgd-login: var(--color-grey-light);
 	--color-bgd-selection: var(--color-primary);
```

Comment:
Wondering if we need these here or at all. Ideally they should get ported up to a more appropriate vars partial?

Reply:
These make sense to move to their respective partial, yes. Will do.